.gitpod.yml

```diff
@@ -0,0 +1,89 @@
+tasks:
+  - name: errors
```

Comment:
We don't need to have separate terminals for `errors` and `api` as they are only internal packages. Logs for this will also be posted in `web` and `js` logs. Please remove these tasks.
Also please change the order so the most important tab (`web`) is at the top 😊

Reply:
Done @mattinannt!